packages/matrix-identity-server/package.json

```diff
@@ -48,6 +48,7 @@
     "express": "^4.19.2",
     "express-rate-limit": "^7.2.0",
     "generic-pool": "^3.9.0",
+    "jest": "^29.7.0",
```

Comment:
Drop this, jest isn't a run dependency and is already declared as dev-dependency into root package.json